cluster/manifests/flannel/daemonset.yaml

```diff
@@ -16,21 +16,33 @@ spec:
     metadata:
       labels:
         application: flannel
-        version: v0.9.1
+        version: v0.10.0
       annotations:
         scheduler.alpha.kubernetes.io/critical-pod: ''
     spec:
       serviceAccountName: system
+      initContainers:
+      - name: install-cni
+        image: registry.opensource.zalan.do/teapot/flannel:v0.10.0
+        command:
+        - cp
+        args:
+        - -f
+        - /etc/kube-flannel/cni-conf.json
+        - /etc/cni/net.d/10-flannel.conflist
```

Comment:
todo: probably better to use the same name as the existing file during the upgrade so that we don't have two configs in the folder.